libmamba/src/api/configuration.cpp

```diff
@@ -1372,6 +1372,16 @@ namespace mamba
                         LockFile timeout for blocking mode when waiting for another process
                         to release the path. Default is 0 (no timeout))")));
 
+        insert(Configurable("disable_lockfile", &ctx.disable_lockfile)
+                   .group("Extract, Link & Install")
+                   .set_rc_configurable()
+                   .set_env_var_names()
+                   .description("Dont use a lockfile")
+                   .long_description(unindent(R"(
+                        Don't create a lockfile when running mamba commands. Use with caution!
+                        -- Some filesystems don't support File Locking and locks don't always
+                        make sense - like when on an HPC.  Default is False (use a lockfile)")));
```

Comment:
Can you add an example of what problems may arise when not using locking? What does Mamba use locks for that will now be the responsibility of the user?

Reply:
Also, what commands are affected by this? If it's only a small selection of commands then it might be worthwhile explicitly listing them here.

Reply:
Depending on how much you want to write on this topic you might also want to link to the new troubleshooting page once it's merged #1873

Reply:
:+1:  I will update this once we decide on the name to use

Reply:
We use lockfiles mainly to lock the package & repodata cache (bad things happen if two processes write to the same file...

We also use it in `micromamba run` to write some json files to store data about the process in `~/.mamba/proc/...`

Reply:
> We also use it in `micromamba run` to write some json files to store data about the process in `~/.mamba/proc/...`

Short question, I have noticed that I can't have more than 1 `micromamba run ...` invocations running at the same time, spawning multiple ones fail with `critical libmamba Another process with name '...' is currently running.` - Similarly if a `micromamba run ...` process get's SIGKILLed, `micromamba run ...` fails with the same error until the zombie `~/.mamba/proc/*.json` get's cleaned up.

Will this PR fix these issues or should I open an issue?

Reply:
@vicecea I think this might be confusing some issues. Running two `run` commands with the same `label` is disallowed. I don't think this PR will fix that. If you can open a new issue and describe exactly what is going on, that would be nice!